modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -3,25 +3,26 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-import com.liferay.commerce.constants.CommerceOrderPaymentConstants;
+package com.liferay.source.formatter.dependencies.upgrade;
 
-import com.liferay.document.library.kernel.model.DLVersionNumberIncrease;
+import com.liferay.commerce.constants.CommerceOrderPaymentConstants;
 
-import com.liferay.document.library.util.DLURLHelperUtil;
+import com.liferay.portal.kernel.util.PortalUtil;
 
-import com.liferay.portal.kernel.search.Sort;
+import com.liferay.document.library.kernel.model.DLVersionNumberIncrease;
+import java.util.Date;
 
 import com.liferay.portal.kernel.service.ServiceContextThreadLocal;
 
 import com.liferay.portal.kernel.util.HtmlParser;
 
-import com.liferay.portal.kernel.util.PortalUtil;
+import com.liferay.portal.kernel.search.Sort;
 
-import java.util.Date;
+import com.liferay.document.library.util.DLURLHelperUtil;
 
-import org.osgi.service.component.annotations.Reference;
+import com.liferay.portal.kernel.cookies.CookiesManagerUtil;
 
-package com.liferay.source.formatter.dependencies.upgrade;
+import org.osgi.service.component.annotations.Reference;
```

Comment:
@MicaelleSilvaa, the previous commit should adjust the imports sorting according to the `replacements.json`, am I right?

Consequently, in this commit, we should just include the import `import com.liferay.portal.kernel.cookies.CookiesManagerUtil;`, does is make sense to you?

If you still have questions about it, please let me know.

Reply:
Resolved 😉